src/Illuminate/Support/Collection.php

```diff
@@ -91,6 +91,17 @@ public function collapse()
     }
 
     /**
+     * Concat the collection with the given items.
+     *
+     * @param  mixed  $items
+     * @return static
+     */
+    public function concat($items)
+    {
+        return new static($this->getArrayableItems($items) + $this->items);
```

Comment:
Why not staying consistant and use `array_merge` as it is done, for `\Illuminate\Support\Collection::merge` ?